input/src/main/java/com/maxkeppeler/sheets/input/type/spinner/SpinnerAdapter.kt

```diff
@@ -0,0 +1,244 @@
+package com.maxkeppeler.sheets.input.type.spinner
+
+import android.content.Context
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.maxkeppeler.sheets.input.R
+import java.lang.ClassCastException
+import java.lang.IllegalStateException
+import java.lang.RuntimeException
+import java.util.*
+
+@Suppress("unused")
+open class SpinnerAdapter(
+    mContext: Context,
+    private var mObjects: MutableList<SpinnerOption> = mutableListOf(),
+    private var mObjectsFromResources: Boolean = false,
```

Comment:
For what is this property used? You only assign it the value false in this adapter.